Assets/MirageXR/Tests/NewUI/ActivityListView_v2.cs

```diff
@@ -363,17 +339,45 @@ public async void MoveAndUnpackTutorialZipFileAndroid(UnityWebRequest www)
 
         File.Delete(savePath);
 
-        TutorialActivityCard(true);
+        CreateTutorialActivityCard();
     }
 
-    public async void MoveTutorialActivityToLocalFilesIOS()
+    private async void MoveTutorialActivityToLocalFilesIOS()
     {
         var stream = new FileStream(Path.Combine(Application.streamingAssetsPath, "TutorialActivity.zip"), FileMode.OpenOrCreate);
 
         await ZipUtilities.ExtractZipFileAsync(stream, Application.persistentDataPath);
 
         stream.Close();
 
-        TutorialActivityCard(true);
+        CreateTutorialActivityCard();
+    }
+
+    private async void CreateTutorialActivityCard()
+    {
+        var t = await TryGetTutorialFromLocalFiles();
+
+        if (t != null)
+        {
+            var sessionContatiner = new SessionContainer { Activity = t };
+
+            _content.Insert(0, sessionContatiner);
+            UpdateView();
+        }
+    }
+
+    private async Task<Activity> TryGetTutorialFromLocalFiles()
+    {
+        var list = await LocalFiles.GetDownloadedActivities();
```

Comment:
I think there is an easier option to load a specific activation. now you load all the local activations and choose one of them